### PR TITLE
Readme content updated

### PR DESCRIPTION
**Pull request by munderseth**
Updates to readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,52 @@
 
-![Testspace Action](https://github.com/testspace-com/setup-testspace/workflows/Testspace%20Action/badge.svg)
+[![Testspace Action](https://github.com/testspace-com/setup-testspace/actions/workflows/ci.yml/badge.svg?branch=readme)](https://github.com/testspace-com/setup-testspace/actions/workflows/ci.yml) 
+
 
 # Testspace client Setup Action
-A GitHub Action is used to install and configure the Testspace client used for publishing test results and reports to [Testspace.com](https://github.com/marketplace/testspace-com). 
+This [GitHub Action](https://github.com/features/actions) is used to install and configure the Testspace client used for publishing test results to [Testspace.com](https://github.com/marketplace/testspace-com).
```
Comment (ivailop) [inline README.md]: The work `used` is repeated twice

File: README.md [hunk 2]
```diff
@@ -1,76 +1,52 @@
...
-      - uses: testspace-com/setup-testspace@v1
-        with:
-          domain: ${{github.repository_owner}}
-      - name: Publish Results to Testspace
-        run: |
-          testspace results.xml
-        if: always()
-```
+- Supports a `branch-based` workflow, and `pull requests`
+- Aggregates results for [jobs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobs), [matrix](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategy), and and even **multiple workflows**
```
Comment (ivailop) [inline README.md]: There are 2 `and` one after another

File: README.md (hunk 1 above)
Comment (munderseth) [reply README.md]: fixed.

File: README.md (hunk 2 above)
Comment (munderseth) [reply README.md]: fixed

**Pull request merged**